lib/install/config/babel.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = function(api) {
         {
           forceAllTransforms: true,
           useBuiltIns: 'entry',
```

Comment:
@gauravtiwari do you think `usage` should be the new default? From [here](https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md):

>Until Babel 7.3, useBuiltIns: usage was unstable and not fully reliable: many polyfills were not included, and many others were added without their required dependencies. In Babel 7.4, I tried to make it understand every possible usage pattern.

it does look like it's stable now. If I understand it correctly, using `usage` won't require additional import. When I use:

```js
import "core-js";
```

in my entry point, I get the following message:

```
When setting `useBuiltIns: 'usage'`, polyfills are automatically imported when needed.
Please remove the direct import of `core-js` or use `useBuiltIns: 'entry'` instead.
```

Reply:
I would question that. `useBuiltIns: 'usage'` is not assuming anything about user's code, just expect to change `import core-js` to required imports based on supported env, so you need to opt-in in into it by adding that import into one of your packs. 
If we switch that to `usage`, it stops being opt-in and starts automatically adding imports to the code. And from my experience, it still has some edge-cases (because core-js detects relevant import by statically analyzing, so it could not predict what you could have in a runtime), so that creates cases when you still need to add some polyfills by hand.

To summarize: I think `useBuiltIns: 'usage'` is a safer choice, in general, that is very easy to change if you know what you're doing.

Reply:
Makes sense 👍